src/Tribe/Repositories/Organizer.php

```diff
@@ -48,26 +48,11 @@ public function __construct() {
 
 		$this->linked_id_meta_key = '_EventOrganizerID';
 
-		$this->schema = array_merge( $this->schema, array(
-			'name' => array( $this, 'filter_by_name' ),
-		) );
-
 		$this->add_simple_meta_schema_entry( 'email', '_OrganizerEmail' );
 		$this->add_simple_meta_schema_entry( 'phone', '_OrganizerPhone' );
 		$this->add_simple_meta_schema_entry( 'website', '_OrganizerWebsite' );
 	}
 
-	/**
```

Comment:
Why remove this?

Reply:
Because that was breaking `tribe_organizers()->where( 'name' )` which uses `post_name` which should be an exact method instead of search.